modules/meshrenderinggl/src/processors/meshrasterizer.cpp

```diff
@@ -1,549 +1,500 @@
+    , baseFrequencyU_("frequencyU", "U-Frequency", 3, 1, 10)
+    , baseFrequencyV_("frequencyV", "V-Frequency", 3, 1, 10)
+    , modulation_("modulation", "Modulation", false)
+    , modulationMode_("modulationMode", "Modulation",
+                      {{"u", "U", HatchingMode::U},
+                       {"v", "V", HatchingMode::V},
+                       {"uv", "UV", HatchingMode::UV}})
+    , modulationAnisotropy_("modulationAnisotropy", "Anisotropy", 0.5f, -1.f, 1.f, 0.01f)
+    , modulationOffset_("modulationOffset", "Offset", 0.f, 0.f, 1.f, 0.01f)
+    , color_("color", "Color", {0.f, 0.f, 0.f})
+    , strength_("strength", "Strength", 0.5f, 0.f, 1.f, 0.01f)
+    , blendingMode_("blending", "Blending",
+                    {{"mult", "Multiplicative", HatchingBlendingMode::Multiplicative},
+                     {"add", "Additive", HatchingBlendingMode::Additive}}) {
+
+    hatching_.getBoolProperty()->setInvalidationLevel(InvalidationLevel::InvalidResources);
+    configComposite(hatching_);
+
+    // init properties
+    color_.setSemantics(PropertySemantics::Color);
+
+    // add to container
+    modulation_.addProperties(modulationMode_, modulationAnisotropy_, modulationOffset_);
+    configComposite(modulation_);
+
+    hatching_.addProperties(mode_, steepness_, baseFrequencyU_, baseFrequencyV_, modulation_,
+                            color_, strength_, blendingMode_);
+
+    baseFrequencyU_.visibilityDependsOn(
+        mode_, [](const auto& prop) { return prop.get() != HatchingMode::V; });
+    baseFrequencyV_.visibilityDependsOn(
+        mode_, [](const auto& prop) { return prop.get() != HatchingMode::U; });
+
+    modulation_.visibilityDependsOn(
+        mode_, [](const auto& prop) { return prop.get() == HatchingMode::UV; });
+}
+
+MeshRasterizer::FaceSettings::FaceSettings(bool frontFace)
+    : frontFace_(frontFace)
+    , show_(frontFace ? "frontcontainer" : "backcontainer", frontFace ? "Front Face" : "Back Face",
+            true)
+    , sameAsFrontFace_("same", "Same as Front Face")
+    , copyFrontToBack_("copy", "Copy Front to Back")
+    , transferFunction_("tf", "Transfer Function")
+    , externalColor_("extraColor", "Color", {1.f, 0.3f, 0.01f})
+    , colorSource_("colorSource", "Color Source",
+                   {{"vertexColor", "VertexColor", ColorSource::VertexColor},
+                    {"tf", "Transfer Function", ColorSource::TransferFunction},
+                    {"external", "Constant Color", ColorSource::ExternalColor}},
+                   2, InvalidationLevel::InvalidResources)
+    , separateUniformAlpha_("separateUniformAlpha", "Separate Uniform Alpha")
+    , uniformAlpha_("uniformAlpha", "Uniform Alpha", 0.5f, 0.f, 1.f, 0.01f)
+    , shadingMode_("shadingMode", "Shading Mode",
+                   {
+                       {"off", "Off", ShadingMode::Off},
+                       {"phong", "Phong", ShadingMode::Phong},
+                       {"pbr", "PBR", ShadingMode::Pbr},

```

Comment:
Hmm, there are multiple PBR-shading methods. We might want to be explicit if more alternatives are to be added?

Reply:
Was actually not implemented in the shader... Removed now.